internal/pkg/template/templates/cicd/pipeline_cfn.yml

```
@@ -124,6 +124,22 @@ Resources:
           {{- end }} {{/* endif ne .Source.OutputArtifactFormat "" */}}{{- end }} {{/* endif ne .Source.ProviderName "GitHubV1" */}}
       Roles:
         - !Ref BuildProjectRole
+  {{- if isSet .Build.AdditionalPolicy }}
```

```suggestion
  {{- if .Build.AdditionalPolicy }}
```
i don't think you need `IsSet`